src/pip/_internal/commands/download.py

```diff
@@ -106,6 +106,10 @@ def run(self, options: Values, args: List[str]) -> int:
 
         reqs = self.get_requirements(args, options, finder, session)
 
+        # Make sure we don't prepare editable metadata.
+        for req in reqs:
+            req.no_build_editable = True
```

Comment:
How do you feel about flipping this; only having this in `install` -- and renaming it to `permit_editable_wheels` or something similar -- to reduce the need to do this in multiple places?

Reply:
Yes, it's better indeed. Done... Although I still think this flag is kind of an intruder on InstallRequirement.